test/odbc/psqlodbc/float.cpp

```diff
@@ -0,0 +1,458 @@
+    expectedInsertLen, DATA_UPDATED_VALUES);
+  dropObject(ServerType::PSQL, "TABLE", TABLE_NAME);
+}
+
+TEST_F(PSQL_DataTypes_Float, Arithmetic_Operators) {
+  const int bufferLen = 0;
+
+  const vector<pair<string, string>> TABLE_COLUMNS = {
+    {COL1_NAME, DATATYPE_NAME + " PRIMARY KEY"},
+    {COL2_NAME, DATATYPE_NAME}
+  };
+
+  const vector<string> INSERTED_PK = {
+    "0.8"
+  };
+
+  const vector<string> INSERTED_DATA = {
+    "22.2"
+  };
+  const int NUM_OF_DATA = INSERTED_DATA.size();
+
+  // insertString initialization
+  string insertString{};
+  string comma{};
+  for (int i = 0; i < NUM_OF_DATA; i++) {
+    insertString += comma + "(" + INSERTED_PK[i] + "," + INSERTED_DATA[i] + ")";
+    comma = ",";
+  }
+
+  const vector<string> OPERATIONS_QUERY = {
+    COL1_NAME + "+" + COL2_NAME,
+    COL1_NAME + "-" + COL2_NAME,
+    COL1_NAME + "*" + COL2_NAME,
+    COL1_NAME + "/" + COL2_NAME,
+    "ABS(" + COL1_NAME + ")",
+    "POWER(" + COL1_NAME + "," + COL2_NAME + ")",
+    "||/ " + COL1_NAME,
+    "LOG(" + COL1_NAME+")"
+  };
+  const int NUM_OF_OPERATIONS = OPERATIONS_QUERY.size();
+
+  vector<vector<double>> expected_results = {};
+
+  // initialization of expected_results
+  for (int i = 0; i < NUM_OF_DATA; i++) {
+    expected_results.push_back({});
+    double data_1 = StringToFloat(INSERTED_PK[i]);
+    double data_2 = StringToFloat(INSERTED_DATA[i]);
+
+    expected_results[i].push_back(data_1 + data_2);
+    expected_results[i].push_back(data_1 - data_2);
+    expected_results[i].push_back(data_1 * data_2);
+    expected_results[i].push_back(data_1 / data_2);
+
+    expected_results[i].push_back(abs(data_1));
+    expected_results[i].push_back(pow(data_1, data_2));
+    expected_results[i].push_back(cbrt(data_1));
+    expected_results[i].push_back(log10(data_1));
+  }
+
+  vector<double> col_results(NUM_OF_OPERATIONS, -1);
+  const vector<long> expectedLen(NUM_OF_OPERATIONS, FLOAT_BYTES_EXPECTED);
+
+  createTable(ServerType::PSQL, TABLE_NAME, TABLE_COLUMNS);
+  insertValuesInTable(ServerType::PSQL, TABLE_NAME, insertString, NUM_OF_DATA);
+  testArithmeticOperators(ServerType::PSQL, TABLE_NAME, COL1_NAME, NUM_OF_DATA, SQL_C_DOUBLE, 
+    col_results, bufferLen, OPERATIONS_QUERY, expected_results, expectedLen);
+  dropObject(ServerType::PSQL, "TABLE", TABLE_NAME);
+}
+
+TEST_F(PSQL_DataTypes_Float, Comparison_Operators) {
+  const vector<pair<string, string>> TABLE_COLUMNS = {
+    {COL1_NAME, DATATYPE_NAME + " PRIMARY KEY"},
+    {COL2_NAME, DATATYPE_NAME}
+  };
+
+  const vector<string> INSERTED_PK = {
+    "0.8"
+  };
+
+  const vector<string> INSERTED_DATA = {
+    "22.2"
+  };
+  const int NUM_OF_DATA = INSERTED_DATA.size();
+
+  // insertString initialization
+  string insertString{};
+  string comma{};
+  for (int i = 0; i < NUM_OF_DATA; i++) {
+    insertString += comma + "(\'" + INSERTED_PK[i] + "\',\'" + INSERTED_DATA[i] + "\')";
```

Comment:
Numbers shouldn't have quotations around them. 
Ditto to all the other places as well.